mods/ts/rules/defaults.yaml

```diff
@@ -609,6 +609,17 @@
 	Tooltip:
 		Name: Palette
 
+^Tunnel:
+	AlwaysVisible:
+	Immobile:
```

Comment:
This leads to fun rendering glitches when the actor's CenterPosition moves outside the viewport:

![tunnel](https://cloud.githubusercontent.com/assets/167819/9596432/a5a75384-506c-11e5-9a81-85a8da5a553d.png)

You'll need to make these Buildings instead.